python/triton/language/standard.py

```diff
@@ -114,11 +112,11 @@ def minimum(x, y, propagate_nan: core.constexpr = PropagateNan.NONE):
 
     .. seealso:: :class:`tl.PropagateNan`
     """
-    return math.min(x, y, propagate_nan)
+    return core.minimum(x, y, propagate_nan)
```

Comment:
@ptillet 
Would it be OK to inline implementation from core here, and call semantic directly?